flowpipe/evaluator.py

```diff
@@ -0,0 +1,235 @@
+"""Classes to evaluate flowpipe Graphs in various ways."""
+
+from concurrent import futures
+import logging
+from multiprocessing import Manager, Process
+import time
+
+
+log = logging.getLogger(__name__)
+
+
+class Evaluator(object):
```

Comment:
Loving it! This was definitely missing, great to see it implemented now. 
We should update the readme and examples as well.

Reply:
Absolutely!